Dockerfile

```diff
@@ -21,17 +21,17 @@ RUN go mod download
 COPY . /build
 RUN make build-release
 
-
-FROM alpine:3.9
-
-RUN apk add --update --no-cache ca-certificates tzdata bash curl
+SHELL ["/bin/bash", "-c"]
 
 # set up nsswitch.conf for Go's "netgo" implementation
 # https://github.com/gliderlabs/docker-alpine/issues/367#issuecomment-424546457
-RUN [ ! -e /etc/nsswitch.conf ] && echo 'hosts: files dns' > /etc/nsswitch.conf
+RUN [ ! -e /etc/nsswitch.conf.build ] && echo 'hosts: files dns' > /etc/nsswitch.conf.build
 
-SHELL ["/bin/bash", "-c"]
+FROM scratch
```

Comment:
How about using a build arg here instead of a completely separate dockerfile for the alpine version?